src/backend/core/user/views.py

```diff
@@ -1,22 +1,23 @@
+from rest_framework import permissions
 from rest_framework.views import APIView
 from rest_framework.response import Response
+from rest_framework_simplejwt.tokens import RefreshToken
 
-from .models import User
 from .serializers import (
     RegistrationSerializer,
     UserSerializer,
-    LoginSerializer,
 )
-from django.conf import settings
-
-import jwt
 
 
 class RegistrationView(APIView):
     """
     View for registering new users.
     """
 
+    # All views have authentication backend by default
```

Comment:
How about AllowAny by default?

Reply:
AllowAny was deleted to assure authentication

Reply:
I see okay, thanks